integration-tests/hibernate-reactive-panache-kotlin/pom.xml

```diff
@@ -0,0 +1,364 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>quarkus-integration-tests-parent</artifactId>
+        <groupId>io.quarkus</groupId>
+        <version>999-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>quarkus-integration-test-hibernate-reactive-panache-kotlin</artifactId>
+    <name>Quarkus - Integration Tests - Hibernate Reactive with Panache and Kotlin</name>
+
+    <!--
+        This is enabled in the Quarkus Maven archetype but not on the integration tests.
+        PanacheQuery.project(Class) needs it.
+    -->
```

Comment:
This comment is irrelevant here (it was for the parameter name property for the compiler)